src/enclave_proc_comm.rs

```diff
@@ -221,14 +215,29 @@ where
             continue;
         }
 
+        let input_stream_raw_fd = events[0].data() as RawFd;
+        let mut input_stream = unsafe { UnixStream::from_raw_fd(input_stream_raw_fd) };
```

Comment:
Why is this safe?

Reply:
This is code that already exists in mainline. However, imo, this is not completely safe since the stream socket peer can close connection (or shut down), making this file descriptor invalid. I'll add an extra check on the EpollEvent received. Thanks!

Reply:
Initially I thought that the file descriptor will be invalidated if the epoll_wait returns an event with error flags set, but apparently it won't be, so this operation is safe